src/test/shared/fakeLogger.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Loggable, Logger } from '../../shared/logger'
+
+// TODO : Consolidate all Test Loggers in a separate change
+export class FakeLogger implements Logger {
```

Comment:
This is a file and class rename from FakeBasicLogger because BasicLogger was renamed to Logger.